packages/fx-core/src/core/tools.ts

```diff
@@ -190,7 +190,11 @@ export function newEnvInfo(
     config: config ?? {
       azure: {},
       manifest: {
-        values: {},
+        values: {
+          appName: {
+            short: "myApp",
```

Comment:
Shall we use the project name to keep consistent with previous behavior? So that it can avoid generating same app name for different project.